zilliqa/src/api/zil.rs

```diff
@@ -1036,6 +1037,11 @@ fn extract_transaction_bodies(block: &Block, node: &Node) -> Result<Vec<Transact
         let receipt = node
             .get_transaction_receipt(*hash)?
             .ok_or(anyhow!("Transaction receipt missing"))?;
+        let receipt_response = TransactionReceiptResponse {
+            cumulative_gas: ScillaGas(receipt.cumulative_gas_used.0).to_string(),
```

Comment:
Shouldn't this be `receipt.cumulative_gas_used.into()`, so the amount gets converted from `EvmGas` to `ScillaGas`?

Reply:
I copied this from the code for GetTxResponse, so I'm guessing it's wrong there too, I'll fix that at the same time

Reply:
Pushed